lib/osf-components/addon/components/contributors/manager/component.ts

```diff
@@ -114,21 +114,27 @@ export default class ContributorsManager extends Component {
         }
     }
 
+    @action
```

Comment:
Why do we have to wrap this in an action?

Reply:
This one was @futa-ikeda's so I'll let him answer it.

Reply:
I think the `@action` decorator here is warranted, but the `(action)` helper in the template file definitely isn't needed. I think I got carried away seeing all the `(perform)s` in there